pytest_jira.py

```diff
@@ -143,8 +144,8 @@ def is_affected(self, issue_id):
         else return False
         """
         return (
-            self._affected_version(issue_id) and
-            self._affected_components(issue_id)
+                self._affected_version(issue_id) and
```

Comment:
this is unnecessary indent.

Reply:
pycharm like it this way apparantly... fixed